inc/main.php

```diff
@@ -44,7 +44,7 @@ function rocket_init() {
 	require WP_ROCKET_FUNCTIONS_PATH . 'files.php';
 	require WP_ROCKET_FUNCTIONS_PATH . 'posts.php';
 	require WP_ROCKET_FUNCTIONS_PATH . 'admin.php';
-	require WP_ROCKET_FUNCTIONS_PATH . 'preload.php';
+	require WP_ROCKET_INC_PATH . '/Engine/Preload/preload.php';
```

Comment:
```suggestion
	require WP_ROCKET_INC_PATH . '/API/preload.php';
```

Reply:
@Tabrisrp Are we using these functions to power parts of Rocket's Engine, i.e. the sourcecode internal to it? Or are they for external themes/plugins/etc to use to trigger things within Rocket?

Reply:
Looks like external stuff. I think you're right. I'll move now.

Reply:
Resolved here https://github.com/wp-media/wp-rocket/pull/2396/commits/8675a6bc19165f5173f4f6d11bbd6e13bcfc005c